pkg/machines/getOSList.py

```diff
@@ -44,6 +53,39 @@
         storage = minimumResources.get_nth(0).get_storage()
         if storage != -1:
             osObj['minimumResources']['storage'] = storage
+
+    osObj['profiles'] = []
+    osInstallScripts = os.get_install_script_list()
```

Comment:
Does this return generator? If not, then you can reuse it on line 62.

Reply:
Done